src/main/java/frc/trigon/robot/subsystems/swerve/SwerveConstants.java

```diff
@@ -103,14 +108,32 @@ public abstract class SwerveConstants {
             TRANSLATION_PID_CONSTANTS.kD
     );
 
-    private static final ReplanningConfig REPLANNING_CONFIG = new ReplanningConfig(true, false);
-    static final HolonomicPathFollowerConfig HOLONOMIC_PATH_FOLLOWER_CONFIG = new HolonomicPathFollowerConfig(
-            AUTO_TRANSLATION_PID_CONSTANTS,
-            AUTO_ROTATION_PID_CONSTANTS,
-            MAX_ROTATIONAL_SPEED_RADIANS_PER_SECOND,
-            SwerveConstants.DRIVE_RADIUS_METERS,
-            REPLANNING_CONFIG
-    );
+    static final RobotConfig ROBOT_CONFIG;
+
+    static {
+        try {
+            ROBOT_CONFIG = RobotConfig.fromGUISettings();
+        } catch (IOException | ParseException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    static final PathFollowingController PATH_FOLLOWING_CONTROLLER = new PathFollowingController() {
+        @Override
+        public ChassisSpeeds calculateRobotRelativeSpeeds(Pose2d currentPose, PathPlannerTrajectoryState targetState) {
+            return null;
+        }
+
+        @Override
+        public void reset(Pose2d currentPose, ChassisSpeeds currentSpeeds) {
+
+        }
+
+        @Override
+        public boolean isHolonomic() {
+            return false;
+        }
+    };
```

Comment:
this is a constants class. Why are these here? And what are tehy Overriding.

Reply:
The new PP update is really weird and confusing. I forgot about all of that stuff